### ISSUE TITLE
Capacitor value?

### ISSUE THREAD

**Issue by acronce** [ghosty-be/esp32-senseair_s8#1]
Thanks for posting your project!

I'm seeing similar erratic measurements with my prototype.

What capacitor value did you use to stabilize the 5 volt supply to the sensor?

**Comment by ghosty-be** [comment]
to be honest I think that the issues were also caused by loose breadboard connections... But I still kept the 470uF 16V electrolytic cap there, just in case... (no specific reason for this value, just something I had in my parts bin salvaged from broken electronics that I scrapped)

**Comment by acronce** [comment]
Thanks for getting back to me. I've also suspected that either loose connections or a frayed wire is contributing to the spikes.

That 470uF cap value seems to be in common use. I've searched and it pops up frequently in ESP projects.

**Comment by acronce** [comment]
I added a 470uf cap (after salvaging it off an old board) and it made a significant difference. Prior to the cap, the sensor output was very jagged. Now it's quite smooth.

Here's a screenshot of the Home Assistant history for the sensor. Before 12 PM on Jan 1 is without the cap. Note that the big dropouts are where I had the sensor off between changes and recalibration.

<img width="1366" alt="Screen Shot 2022-01-02 at 7 58 34 AM" src="https://user-images.githubusercontent.com/61846960/147881636-becb97ad-9419-4e4e-a706-01a353c8555d.png">

In case anyone is interested in trying this, you will need to recalibrate after adding the cap. I found that with the cap in place the sensor was outputting double digit values (very low). At first I thought I damaged the sensor. But after recalibration it seems to work fine and is providing sensible results for indoor CO2 levels.
